Tests/New_Customer/test_new_customer.py

```diff
@@ -8,7 +7,9 @@
 @pytest.mark.usefixtures("setup")
 class TestNewCustomer:
 
-    def test_new_customer_input_message(self):
+    # @pytest.mark.parametrize("expected_message", [("Customer Name,", "Customer name must not be blank")])
+    @pytest.mark.parametrize("expected_message", [("Customer Name", "Customer name must not be blank"), ("City", "City Field must be not blank"), ("State", "State must not be blank"), ("PIN", "PIN Code must not be blank"), ("Phone_number", "Telephone no must not be blank"), ("Email", "Email-ID must not be blank")])
+    def test_new_customer_input_message(self, expected_message):
```

Comment:
As per my comment below, may you need to change it to this:
```suggestion
    @pytest.mark.parametrize("input_name, expected_message", [("Customer Name", "Customer name must not be blank"), ("City", "City Field must be not blank"), ("State", "State must not be blank"), ("PIN", "PIN Code must not be blank"), ("Phone_number", "Telephone no must not be blank"), ("Email", "Email-ID must not be blank")])
    def test_new_customer_input_message(self, input_name, expected_message):
```